Update docs/adfs/README.md and move to wiki

1nbuc opened the pull request:
# Description

- Added Powershell command to activate signing on message and aseertion on AD FS in docs/adfs/README.md
- Added Link to Microsoft Documentation with a list of all available authnContext options.

# Checklist:

- Issue Addressed: [ ]
- Link to SAML spec: [ ]
- Tests included? [ ]
- Documentation updated? [X]

cjbarth commented on docs/adfs/README.md:
```diff
@@ -57,6 +58,8 @@ passport.use(
       // From the metadata document
       audience: "https://adfs.acme_tools.com/FederationMetadata/2007-06/FederationMetadata.xml",
+      // otherwise it will thow because of invalid document signature
+      wantAuthnResponseSigned: false,
```
We don't want any example code to disable security features.

1nbuc replied:
But it is very confusing  since this  is the exact configuration for AD FS and not working out of the box.
If possible it could also be mentioned how to adjust the ADFS configuration to work with it but for me there was no other oppertunity than setting `wantAuthnResponseSigned`to false

1nbuc commented:
Honestly I didn't know about the possibility to change the signing behaviour, I added the powershell script to the docs and removed the insecure configuration.

cjbarth commented:
@1nbuc , for record-keeping purposes, can you please adjust the description of this PR to follow the directions?

cjbarth commented:
Can you `npm run prettier-format` on this code so that tests pass?

cjbarth commented:
This document has been moved to the Wiki.
